frontend/app/src/people/widgetViews/OrganizationDetails.tsx

```diff
@@ -383,14 +385,14 @@ const OrganizationDetails = (props: { close: () => void; org: Organization | und
           <Button
             disabled={addWithdrawDisabled}
             text="Withdraw"
-            color="white"
+            color="withdraw"
```

Comment:
is this even a color?

Reply:
@kevkevinpal went with since we have the likes of 
https://github.com/stakwork/sphinx-tribes/blob/50cf7837840a869c7c9f7f8b17c6daa1356d066c/frontend/app/src/config/commonColors.ts#L15-L41
in the same `commonColors` file.

Reply:
ahh gotcha